feature/settings/build.gradle.kts

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation(libs.androidx.material3)
     implementation(libs.androidx.hilt.navigation.compose)
     implementation(libs.hilt.android)
+    implementation(project(":design"))
```

Comment:
Can we move this to the top of the dependencies block next to the analytics project and reference it in the same way i.e.
implementation(projects.design)